examples/xml-from-record-conversion/xml_from_record_conversion.bal

```diff
@@ -0,0 +1,41 @@
+import ballerina/io;
+import ballerina/xmldata;
+
+// Defines a record type with annotations.
+@xmldata:Namespace {
+    prefix: "ns",
+    uri: "http://sdf.com"
+}
+type Invoice record {
+    int id;
+    Item[] items;
+    @xmldata:Attribute
+    string 'xmlns?;
```

Comment:
Normally this xmlns value is knowns when defining the record right? it is not changed in value. Shall we add the value here